src/main/scala/com/danieljharvey/eggserver/LevelsLoader.scala

```diff
@@ -19,4 +63,42 @@ object LevelsLoader {
         MySQL.getLevel(levelID)
     }
 
+    def decodeLevelData (json: String): JSONLevelData = {
+        implicit val formats = DefaultFormats
+        parse(json).extract[JSONLevelData]
+    }
+
+    def tidyLevel (levelData: JSONLevelData): LevelData = {
+        val newBoard = levelData.board.map(row => row.map(jsonTile => {
+            tidyTile(jsonTile)
+        }))
```

Comment:
If you ever-so wish to tersen this up, then: `levelData.board.map(_.map(tidyTile))` ;)